tools/markdown-magic/src/transforms/installationInstructionsTransform.cjs

```diff
@@ -3,28 +3,36 @@
  * Licensed under the MIT License.
  */
 
+const { defaultSectionHeadingLevel } = require("../constants.cjs");
 const {
 	formattedGeneratedContentBody,
 	formattedSectionText,
 	getPackageMetadata,
 	resolveRelativePackageJsonPath,
+	parseHeadingOptions,
 } = require("../utilities.cjs");
 
 /**
  * Generates a simple Markdown heading and contents with package installation instructions.
  *
  * @param {string} packageName - Name of the package (fully scoped).
  * @param {boolean} devDependency - Whether or not the package is intended to be installed as a dev dependency.
- * @param {boolean} includeHeading - Whether or not to include the heading in the generated contents.
+ * @param {object} headingOptions - Heading generation options.
+ * @param {boolean} headingOptions.includeHeading - Whether or not to include a top-level heading in the generated section.
+ * @param {number} headingOptions.headingLevel - Root heading level for the generated section.
+ * Must be a positive integer.
  */
-const generateInstallationInstructionsSection = (packageName, devDependency, includeHeading) => {
+const generateInstallationInstructionsSection = (packageName, devDependency, headingOptions) => {
 	const sectionBody = `To get started, install the package by running the following command:
 
 \`\`\`bash
 npm i ${packageName}${devDependency ? " -D" : ""}
 \`\`\``;
 
-	return formattedSectionText(sectionBody, includeHeading ? "Installation" : undefined);
+	return formattedSectionText(sectionBody, {
+		...headingOptions,
+		headingText: "Installation",
```

Comment:
[data-structures/map](https://github.com/Josmithr/FluidFramework/blob/docs/up-markdown-documenter/docs/content/docs/data-structures/map.md)

Found one with `h3` but I believe this is acceptable as the tag lives inside the `<h2>` tag.

Reply:
That document isn't using these templates. It's also a bit different since it's public-user-facing, and is pointing users to the `fluid-framework` package.